Makefile

```diff
@@ -168,6 +168,7 @@ run-tests-local-process: build test-images
 		--name=$(TESTCONTAINER) \
 		-v $(ROOTDIR):/usr/code \
 		-e CGO_ENABLED=0 \
+		-e GOCACHE=off \
```

Comment:
Out of interest: why switch this cache off?

Reply:
In go 1.10 `go test` will cache its result and re-use when there are no code changes.